common/src/main/java/com/wynntils/services/chat/ChatTabService.java

```diff
@@ -81,6 +81,12 @@ public int getNextFocusedTab() {
         return (getTabIndex(getFocusedTab()) + 1) % getTabCount();
     }
 
+    public int getLastFocusedTab() {
+        int tabCount = getTabCount();
+        int lastTab = getTabIndex(getFocusedTab()) - 1;
+        return lastTab == -1 ? tabCount - 1 : lastTab;
```

Comment:
```suggestion
    int tabIndex = getTabIndex(getFocusedTab());
    return (tabIndex - 1 + getTabCount()) % getTabCount();
```

I think this expresses the edge-case where only 1 tab is in the list better (and since you are already changing the name, this is not a "hassle").